.gitignore

```diff
@@ -29,8 +29,11 @@ pubspec.lock
 build/
 
 # ignore generated files
-gen/
+**/gen/**
 *.mocks.dart
 
+# Do not ignore generated version
+!**/gen/version.dart
+
```

Comment:
Why? It's generated code and is not supposed to be checked in. It should be generated when publishing a new version, at least that's the idea

Reply:
My bad. While executing the code we got an error that version.dart was missing and this solved the problem at the time.